ibm_catalog.json

```diff
@@ -392,7 +392,110 @@
                 "description": "This architecture supports creating and configuring IBM Cloud Logs resources"
               }
             ]
-          }
+          },
+          "dependencies": [
+            {
+              "name": "deploy-arch-ibm-cos",
+              "description": "Cloud automation for Cloud Object Storage supports creating an IBM Cloud Object Storage instance, cross-region and single-region bucket.",
+              "id": "68921490-2778-4930-ac6d-bae7be6cd958-global",
+              "version": "v8.21.25",
+              "flavors": [
+                "instance"
+              ],
+              "catalog_id": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3",
+              "optional": true,
+              "on_by_default": true,
+              "input_mapping": [
+                {
+                  "dependency_output": "cos_instance_crn",
+                  "version_input": "existing_cos_instance_crn"
+                },
+                {
+                  "dependency_input": "prefix",
+                  "version_input": "prefix",
+                  "reference_version": true
+                },
+                {
+                  "dependency_input": "use_existing_resource_group",
+                  "value": true,
+                  "reference_version": true
+                },
+                {
+                  "dependency_input": "resource_group_name",
+                  "version_input": "existing_resource_group_name",
+                  "reference_version": true
```

Comment:
In v9 of COS DA, `use_existing_resource_group` and `resource_group_name` are replaced by `existing_resource_group_name`